cookiestore/cookieListItem_attributes.https.any.js

```diff
@@ -109,15 +114,10 @@ promise_test(async testCase => {
   const cookie = await cookieStore.get('cookie-name');
   assert_equals(cookie.name, 'cookie-name');
   assert_equals(cookie.value, 'cookie-value');
-  assert_equals(cookie.domain, kCurrentHostname);
-  assert_equals(cookie.path, '/');
-  assert_equals(cookie.expires, null);
-  assert_equals(cookie.secure, true);
-  assert_equals(cookie.sameSite, 'strict');
-  const itemKeys = Object.keys(cookie);
-  for (const key of kCookieListItemKeys) {
-    assert_in_array(key, itemKeys);
-  }
+  assert_cookie_keys(cookie);
+
+  const internalCookie = await test_driver.get_named_cookie('cookie-name');
+  assert_equals(internalCookie.domain, kCurrentHostname);
```

Comment:
In Firefox, this fails because the domain is `.web-platform.test`. It's also possible that this is a bug in our WebDriver BiDi implementation. I'll check.

Reply:
Yeah, we have a similar thing where internally we store it with a leading dot and then need to undo that.